Dockerfile

```diff
@@ -0,0 +1,22 @@
+#syntax=docker/dockerfile:1
```

Comment:
if possible move to ./docker/dockerfiles

Reply:
or, how should it be called?

Reply:
Is it meant as a replacement for stage2? If so stage2.

Reply:
@sami-sweng it's meant to replace the whole `docker` folder.

Only replacing stage2 seems to make stage1 moot. Does it sound correct?

Reply:
The stage 1 will still be used in the CI and there is the dev one.

Let's keep it, there is never only one dockerfile.

Reply:
> Let's keep it, there is never only one dockerfile.

let's call it. `docker/dockerfiles/Dockerfile` then.